test/test_checking.cpp

```diff
@@ -329,4 +333,3 @@ TORRENT_TEST(force_recheck)
 {
 	test_checking(force_recheck);
 }
-
```

Comment:
do you get warnings for these?

Reply:
no, that's me again removing _extra_ '\n's